tests/formatter.rs

```diff
@@ -1690,3 +1691,424 @@ help: consider removing the `?Sized` bound to make the type parameter `Sized`
     let renderer = Renderer::plain();
     assert_data_eq!(renderer.render(input_new), expected);
 }
+
+#[test]
+fn e0271() {
```

Comment:
what are these test names?

Can we add these tests without unicode support and then show how unicode support changes them?